src/Files.App/ViewModels/SettingsViewModel.cs

```diff
@@ -39,28 +39,19 @@ public SettingsViewModel()
             UpdateThemeElements = new RelayCommand(() => ThemeModeChanged?.Invoke(this, EventArgs.Empty));
         }
 
-        public async Task DetectQuickLook()
+        public Task DetectQuickLook()
         {
             // Detect QuickLook
             try
             {
-                var connection = await AppServiceConnectionHelper.Instance;
-                if (connection != null)
-                {
-                    var (status, response) = await connection.SendMessageForResponseAsync(new ValueSet()
-                    {
-                        { "Arguments", "DetectQuickLook" }
-                    });
-                    if (status == AppServiceResponseStatus.Success)
-                    {
-                        App.AppModel.IsQuickLookSupported = response.Get("IsAvailable", false);
-                    }
-                }
+                App.AppModel.IsQuickLookSupported = QuickLookHelpers.CheckQuickLookAvailability();
```

Comment:
Do you want to create a new startup task for this (should be delayed until everything else is loaded).

Reply:
Yeah, I can do that. Just put it in the QL helper class?

Reply:
I think that's good for now 👍

Reply:
Is there an await missing on this line?

Reply:
It wasn't at the time, but the methods have been changed, so I've added in.